ADD_NEW_DISTRO.md

```diff
@@ -0,0 +1,21 @@
+# How to add a new distro
+
+## Add Dockerfiles
+
+Create `build_<DISTRO>.dockerfile` and `run_<DISTRO>.dockerfile` in the root directory.
+
+* These are based on [multi-stage builds](https://docs.docker.com/develop/develop-images/multistage-build/).
+* Only install the needed dependencies before the GHDL compilation/installation.
```

Comment:
"Images should contain the dependencies for building or installing GHDL only. Do not include the actual building/installation procedures in the dockerfiles".